common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp

```diff
@@ -68,6 +68,8 @@ class Logger : public nvinfer1::ILogger  // NOLINT
     }
   }
 
+  void log(Severity severity, const std::string & msg) noexcept { log(severity, msg.c_str()); }
```

Comment:
This logger is passed to TensorRT API, so you should not use it to output log. You can call rclcpp logger macro directly.

Reply:
I understand, but was asked to use it this way:

> Cloud you try to use `logger_`? It's a wapper of ros message for tensort library. https://github.com/autowarefoundation/autoware.universe/blob/main/common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp#L38-L73

Reply:
In the same way, I did not use RCLCPP from the beginning since the caller of the method I was using, was already using `std::cout` (-> code consistency). 
Should I just RCLCPP in all the changes in this commit  @yukke42

Reply:
@knzo25 Cloud you follow @wep21's proposal? :bow: